website/screens/principles/iconography/usage/IconographyUsagePage.tsx

```diff
@@ -0,0 +1,97 @@
+import DocFooter from "@/common/DocFooter";
+import QuickNavContainer from "@/common/QuickNavContainer";
+import QuickNavContainerLayout from "@/common/QuickNavContainerLayout";
+import { DxcBulletedList, DxcFlex } from "@dxc-technology/halstack-react";
+import Image from "@/common/Image";
+import icons from "./images/Iconography_image.png";
+import Link from "next/link";
+
+const sections = [
+  {
+    title: "Usage",
+    content: (
+      <>
+        <Image src={icons} height={400} alt="Halstack Icons" />
+        <DxcBulletedList>
+          <DxcBulletedList.Item>
+            Icon sizes can vary based on use case and application context. Most
+            of the icons that are currently used within Halstack components
+            follow a 24x24px size including the background or space around the
+            icon. There are a few exceptions where 20x20 or 16x16 icon sizes are
+            used as these sizes better match the visual context of the element.
+          </DxcBulletedList.Item>
+
+          <DxcBulletedList.Item>
+            Currently, the default color of icons is always inherited from the
+            parent during implemented. If no color is specified, the icon will
+            black by default (with a few cases where the default is purple).
+          </DxcBulletedList.Item>
+          <DxcBulletedList.Item>
+            Icons come in outlined and filled variants. The currently
+            implemented icons use base colors of black or white for visual
+            consistency.
+          </DxcBulletedList.Item>
+          <DxcBulletedList.Item>
+            While there currently exists a library of over three thousand icons,
+            there may be instances when a custom icon is needed (such as icons
+            that represent brands, companies, or organizations), an SVG
+            implementation option is also available for use. This use should be
+            limited only to these situations as it is highly encouraged to use
+            the existing icon set for design consistency.
+          </DxcBulletedList.Item>
+        </DxcBulletedList>
+      </>
+    ),
+  },
+  {
+    title: "Best Practices",
+    content: (
+      <>
+        <DxcBulletedList>
+          <DxcBulletedList.Item>
+            Use icons within the context of commonly used components such as
+            buttons, links, navigation items, or status indicators.
+          </DxcBulletedList.Item>
+          <DxcBulletedList.Item>
+            Do not use icons that fill the entire space allotted for the icon.
+            All icons currently implemented maintain a minimum of 4px padding
+            (the largest solid icon occupies a maximum of 20px width or height).
```

Comment:
Oh I meant the symbol itself for example the menu icon:

![image](https://github.com/dxc-technology/halstack-react/assets/101174179/a24f2550-bce3-43f4-bc56-0e83b5a2e892)

The three solid lines occupy a maximum of 20px while leaving 4px minimum white space all around so that the symbol does not touch the edge of the 24px space

Reply:
Ahh.... the new material symbols library is much larger than the icons I was looking at when I wrote this... yes... so we have a minimum of 2px white space between the actual symbol and the edge of the icon area